src/Forms/ElementalAreaField.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace DNADesign\Elemental\Forms;
+
+use SilverStripe\Forms\GridField\GridField;
+use SilverStripe\Forms\CompositeField;
+use SilverStripe\Forms\FieldGroup;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\TabSet;
+use SilverStripe\Forms\ReadonlyTransformation;
+
+class ElementalAreaField extends GridField
+{
+    public function performReadonlyTransformation()
+    {
+        $readOnlyField = $this->castedCopy(CompositeField::class);
+        
+        $blockReducer = function ($element) {
```

Comment:
Could be separated into a new method, e.g. `protected function getBlockReducer() : callable`